packages/blade-old/src/atoms/Badge/Badge.native.js

```diff
@@ -62,6 +67,8 @@ const styles = {
           return 'neutral.900';
         case 'neutral':
           return 'primary.900';
+        case 'cyan':
+          return 'tertiary.900';
```

Comment:
If variant is `cyan` and `fill` is `solid`, you'll end up having background as well as foreground as `tertiary.900`, right?

Reply:
In that case, it'll be `primary.900`
![image](https://user-images.githubusercontent.com/43565572/128133994-2f757888-8ef4-4fc1-b5a2-be0667b86aee.png)